test/ctia/entity/web_test.clj

```diff
@@ -403,15 +401,14 @@
           (jwt-http-checks-test
            url-1
            url-2
-           (fn [{:keys [get-judgement jwt-1 jwt-2]}]
+           (fn [{:keys [get-judgement jwt-1]}]
              (testing "Check URL server returns 401"
                (is (= 401 (:status (get-judgement jwt-1))))
-               (is (= {:error "invalid_jwt"
+               (is (= {:error :invalid_jwt
                        :error_description
                        (str "(56bb5f8c-cc4e-4ed3-a91a-c6604287fe32) SERVER ERROR DESCRIPTION: Bearer "
                             jwt-1)}
-                      (json/read-str (:body (get-judgement jwt-1))
-                                     :key-fn keyword))
+                      (:parsed-body (get-judgement jwt-1)))
```

Comment:
This seems to be the only place the jwt invalidation code is exercised. The json format is no longer hard-coded, and is actually EDN thanks to the defaults of clj-momo's GET.

Reply:
Might be good to also exercise json.

Also I'm not sure that leveraging clj-momo's GET default yields a clear test.

A test would ideally exemplify a typical usage. If typical usage is json, then reflecting EDN as the default can result in  a test that can break later (and importantly, leave people unsure of what the right fix would be)

Reply:
Thanks, it took me a week to think of the best way to achieve that but this explicits tests both json and edn https://github.com/threatgrid/ctia/pull/1083/commits/5aef81dca784523aab8830f40e62c47a1d414680

Reply:
@vemv PR ready for another look.